docs/build-guide.rst

```diff
@@ -27,35 +27,20 @@ add instructions for any other distributions.
 NixOS Linux
 ^^^^^^^^^^^
 
-First of all, install ``mono``, ``fsharp`` and ``dotnetPackages.Nuget``::
+*The instructions has been verified on NixOS 16.03. If something
+ doesn't work, please file an issue.*
```

Comment:
It seems that we have a problem with this paragraph.See how it was rendered: https://github.com/codingteam/naggum/blob/feature/nix-shell-71/docs/build-guide.rst#nixos-linux

Maybe the space at the second line is the problem?